internal/testutils/ocm_mock.go

```diff
@@ -0,0 +1,61 @@
+package testutils
+
+import (
+	"context"
+
+	amv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+// NewMockOCMClient returns a *MockOCMClient configured with the supplied options.
+func NewMockOCMClient(opts ...MockOCMClientOption) *MockOCMClient {
```

Comment:
If the type satisfies the interface, would we not want to return `types.OCMClient` instead?